tests/support/server.tcl

```diff
@@ -436,13 +441,17 @@ proc start_server {options {code undefined}} {
     set args {}
     set keep_persistence false
     set config_lines {}
+    set start_old_server 0
 
     # Wait for the server to be ready and check for server liveness/client connectivity before starting the test.
     set wait_ready true
 
     # parse options
     foreach {option value} $options {
         switch $option {
+            "start-old-server" {
+                set start_old_server $value
+            }
```

Comment:
Don't clearly get how does this option pick which old server to start.

Reply:
It's just a boolean (1 or 0), to start the old server or to start the normal server.

The server to start is a global variable `$::old_server_path`, set by the `test_helper.tcl` script when parsing the `--old-server-path` option.

Reply:
Added a comment. (And changed to "other".)